lib/features/authentication/login/page/login.dart

```diff
@@ -14,10 +13,8 @@ class LoginPage extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
-    return Scaffold(
-      resizeToAvoidBottomInset: false,
-      appBar: AppBar(),
```

Comment:
the app bar is needed here so user can go back, 
i know it's not on the design but i decided it's better to have it there